### PR TITLE
Create Emilia - Redone

### PR DESCRIPTION
**Pull request by Alcala09**
This is redone so it doesn't replace anybody

### CODE REVIEW

**Comment by yourWaifu** [comment]
can you put the .json to the end, also you don't need to do a pull request just push changes the branch

**Comment by yourWaifu** [comment]
also I just added the hacktoberfest tag, can you help me make sure it's counting your pull request?

**Comment by Alcala09** [comment]
> also I just added the hacktoberfest tag, can you help me make sure it's counting your pull request and how many times it counted?

It has requested for Emilia twice, the second one is the improved one with the changes you requested. For Mai Sakurajima, I do not see her in the main pull request page.

**Comment by Alcala09** [comment]
Are there any other issues with the JSON script?

**Comment by yourWaifu** [comment]
it's not a script, but I'll look over it like usally, I think I need an automated tool for this.

**Comment by yourWaifu** [comment]
ok here comes the screenshots 
image
image

I'll point out the issues.

File: waifus/emilia.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+	"type" : 0,
+	"english-name" : "emilia",
+	"image" : "https://vignette.wikia.nocookie.net/p__/images/6/60/Emilia_Infobox.png/revision/latest?cb=20200812110513&path-prefix=protagonist",
+	"japanese-name" : "エミリア",
+	"year" : ,
```
Comment (yourWaifu) [inline waifus/emilia.json:6]: If the year is unknown, remove this line otherwise, it has to be a number.

File: waifus/mai sakurajima.json [hunk 2]
```diff
@@ -0,0 +1,11 @@
+{
+	"type" : 0,
+	"english-name" : "mai sakurajima",
+	"image" : "https://i.pinimg.com/474x/34/5a/88/345a8893bd6dd27b3407be37ee50a885.jpg",
+	"japanese-name" : "桜島 麻衣",
+	"year" : ,
```
Comment (yourWaifu) [inline waifus/mai sakurajima.json:6]: same goes for this one.

**Review by yourWaifu — CHANGES_REQUESTED** [review]

File: waifus/mai sakurajima.json (hunk 2 above)
Comment (Alcala09) [reply waifus/mai sakurajima.json]: Alright I removed the year lines

**Comment by yourWaifu** [comment]
more screenshots 
image
image

**Comment by yourWaifu** [comment]
There's a few things I want you to add to the notes, it says that she was frozen at one point, I feel that's an important detail that describes Emilia's age.

File: waifus/emilia.json [hunk 3]
```diff
@@ -0,0 +1,10 @@
+{
+	"type" : 0,
+	"english-name" : "emilia",
```
Comment (yourWaifu) [inline waifus/emilia.json:3]: Can you make the first letter capital

File: waifus/mai sakurajima.json [hunk 4]
```diff
@@ -0,0 +1,10 @@
+{
+	"type" : 0,
+	"english-name" : "mai sakurajima",
```
Comment (yourWaifu) [inline waifus/mai sakurajima.json:3]: Same here

**Review by yourWaifu — CHANGES_REQUESTED** [review]

**Comment by Alcala09** [comment]
Alright made a note about Emilia being frozen for 100 years, and capitalized the names (not for the json)

**Review by yourWaifu — APPROVED** [review]
Looks good

**Pull request merged**